odps-data-carrier/task-scheduler/src/main/java/com/aliyun/odps/datacarrier/taskscheduler/Action.java

```diff
@@ -20,20 +20,23 @@
 package com.aliyun.odps.datacarrier.taskscheduler;
 
 public enum Action {
```

Comment:
Add odps drop table and odps drop partition, so that there won't be multiple sqls within a single action